spec/services/usda_service_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe 'usda service' do
```

Comment:
Per your comment, yes, more robust testing would be good. And in the Service and Facade we can test for data types.